.rubocop.yml

```diff
@@ -5,7 +5,7 @@ Rails:
   Enabled: true
 
 AllCops:
-  TargetRubyVersion: 2.3
+  TargetRubyVersion: 2.4
```

Comment:
FYI. this is not a change in the ruby version but instead a change in the rubocop rules to reflect the ruby version we were already using.

Reply:
👍 thanks for the FYI